Update/yoig_010.txt

```diff
@@ -3822,7 +3822,7 @@ void main()
 	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#aa886c>美雪</color>", NULL, "<color=#aa886c>Miyuki</color>", NULL, Line_ContinueAfterTyping); }
 	ModPlayVoiceLS(4, 40, "ps3/s10/40/410300902", 256, TRUE);
 	OutputLine(NULL, "「だから、彼に対してその文句や不満を言って嫌われたりしたら、彼は自分を必要としなくなって、手元から離れていくかもしれない」",
-			NULL, "\"Because you were worried that if you ever complained to him or told him how you really felt, he might walk out on you.\"", Line_Normal);
+			NULL, "\"Because you were worried that if you ever complained to him or told him how you really felt, he might walk out on you, realizing he was no longer needed.\"", Line_Normal);
```

Comment:
```suggestion
			NULL, "\"Because you were worried that if you ever complained to him or told him how you really felt, he might realize he was no longer needed and walk out on you.\"", Line_Normal);
```